### ISSUE TITLE
Java SE 6 compliance

### ISSUE THREAD

**Issue by jferard** [iryndin/jdbf#28]
The source makes a very little use of Java SE 7 features, so why don't make a Java SE 6 compliant code?
I know that Java SE 6 was released on December 11, 2006. So Java SE 6 is almost ten years old. But some companies still use it (see https://plumbr.eu/blog/java/java-version-and-vendor-data-analyzed-2016-edition: 9.56% for Java SE 6). 
I need a dbf library that works with Java SE 6, and I noticed that there are only two diamonds operators in the sources, and three try with resources statements in the tests. It took me five minutes to make jdbf code compliant with Java SE 6.
To sum up :
1. around 10% of Java users use Java SE 6;
2. two lines in the source to fix.
So, why not?

**Comment by iryndin** [comment]
@jferard Sorry, there are no plans to make it `Java 6` compliant. You better make a fork and make your fork `Java 6` compliant.
